packages/cli/test/fixture-projects/sample-project/contracts/modules/UpgradeModule.sol

```diff
@@ -1,9 +1,9 @@
 //SPDX-License-Identifier: MIT
 pragma solidity ^0.8.0;
 
-import "@synthetixio/core-modules/contracts/modules/CoreUpgradeModule.sol";
+import {UpgradeModule as BaseUpgradeModule} from "@synthetixio/core-modules/contracts/modules/UpgradeModule.sol";
```

Comment:
```suggestion
import { UpgradeModule as BaseUpgradeModule } from "@synthetixio/core-modules/contracts/modules/UpgradeModule.sol";
```

Reply:
@ajsantander actually, the linter says that it is an error adding those spaces, looks like its not a common practice on Solidity, even the docs doesn't use them: https://docs.soliditylang.org/en/v0.5.0/layout-of-source-files.html?highlight=import#import

Reply:
Okay, closing then